src/main/java/hudson/plugins/msbuild/MsBuildKillingVeto.java

```diff
@@ -49,17 +52,19 @@ public class MsBuildKillingVeto extends ProcessKillingVeto {
     * 
     */
     @Override
+    @SuppressFBWarnings(value = "RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE", justification = "proc.getArguments() could be null or empty")

```

Comment:
https://javadoc.jenkins.io/hudson/util/ProcessTreeRemoting.IOSProcess.html#getArguments() shows that the proc.getArguments() return value is annotated with `NonNull`.  Spotbugs uses that to remind that the `cmdLine == null` can be removed from the later conditional.

Could you remove that null check so that this suppression is not needed?
```suggestion
```